test/generators/suspenders/install/web_generator_test.rb

```diff
@@ -12,6 +12,14 @@ class WebGeneratorTest < Rails::Generators::TestCase
         setup :prepare_destination
         teardown :restore_destination
 
+        test "generates Gemfile entry with version constraint" do
+          with_database "postgresql" do
+            run_generator
+          end
+
+          assert_file "Gemfile", /gem "suspenders", "> 20240101"/
+        end
```

Comment:
Were you able to get this to pass locally? I'm having some local dependency issues, so I can't verify this locally.

That being said, the file we're modifying in this commit is `lib/install/web.rb`, which is not a generator, but the application template.

This test is  responsible for testing [`Suspenders ::Generators::Install:: WebGenerator`](https://github.com/thoughtbot/suspenders/blob/include-gemfile-version/lib/generators/suspenders/install/web_generator.rb), which doesn't install Suspenders.

Reply:
Unfortunately, we don't have any coverage for `lib/install/web.rb`.